src/main/java/io/github/jhipster/online/security/jwt/TokenProvider.java

```diff
@@ -121,7 +121,7 @@ public Authentication getAuthentication(String token) {
 
     public boolean validateToken(String authToken) {
         try {
-            Jwts.parser().setSigningKey(key).parseClaimsJws(authToken);
+            Jwts.parserBuilder().setSigningKey(key).build().parseClaimsJws(authToken);
```

Comment:
Same thing as above

Reply:
Also current jhipster-dependencies uses jjwt 0.11.2 rather than 0.11.0 from the version 3.6.0. Although I have no idea about what it takes to upgrade jhipster-online nor if it's an objective for JHO to keep up with JHipster versions.

0.11.2 fixes a CVE in Jackson, https://github.com/jwtk/jjwt/blob/master/CHANGELOG.md

Reply:
@gmarziou : As for the upgrading the jhipster-dependencies version we point to; I think we normally do it when we upgrade jhipster-online. I'll try to upgrade jhipster-online when we release v7. 😄

Reply:
@gmarziou : Although the application works fine I think this was done that way because of the tests maybe. The tests seems to fail as when the constructor is invoked the values are null. Is there a way where we could set the values while initializing the object in junit? I tried some ways but I couldn't still figure out that part. 🤔